pkg/virt-handler/device-manager/mediated_device.go

```diff
@@ -230,7 +228,7 @@ func (dpi *MediatedDevicePlugin) register() error {
 	reqt := &pluginapi.RegisterRequest{
 		Version:      pluginapi.Version,
 		Endpoint:     path.Base(dpi.socketPath),
-		ResourceName: dpi.resourceName,
+		ResourceName: dpi.deviceName,
```

Comment:
Why do we change in using the deviceName instead of resourceName?

Reply:
The 2 fields `resourceName` and `deviceName` had the same value that was written only once. I couldn't find any difference between them, so I merged them.
Do you know why these 2 fields were separate?

Reply:
@akrejcir good question but I don't know what was the original intention. Maybe @vladikr knows? Anyway, make totally sense to use a single variable

Reply:
I'm not sure too, tbh. I think this is a result of a merge of several device plugins that we used to have. It makes sense to just go with a single variable. Perhaps we should use the name `resourceName` instead? resource name would be something like `nvidia.com/GRID_T4-1Q`

Reply:
Renamed.